internal/broker/export_test.go

```diff
@@ -105,5 +107,25 @@ func (b *Broker) FetchUserInfo(sessionID string, cachedInfo *authCachedInfo) (st
 		return "", err
 	}
 
-	return b.userInfoFromClaims(uInfo, groups)
+	userInfo := b.userInfoFromClaims(uInfo, groups)
+	data, err := json.Marshal(userInfo)
+	if err != nil {
+		return "", err
+	}
+	var indented bytes.Buffer
+	err = json.Indent(&indented, data, "", "\t\t")
+	return indented.String(), err
+}
+
+func init() {
+	defaultStringifyJSON := stringifyJSON
+	stringifyJSON = func(b []byte, err error) (string, error) {
+		if err != nil {
+			return defaultStringifyJSON(b, err)
+		}
+		// Indent in testing, to improve readability of golden files
+		var indented bytes.Buffer
+		err = json.Indent(&indented, b, "", "  ")
+		return defaultStringifyJSON(indented.Bytes(), err)
+	}
```

Comment:
I'm sorry, @3v1n0, but I still don't like doing this just for readability. I think our golden files should match the actual output of the function as close as we can.

Reply:
Ok, we can also use this in production code, but if you don't care about readability (I did it only because I thought it was a project goal, and to help you and others reviewing the PR), I'll just drop this.